projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/write-linked-cases-field.component.ts

```diff
@@ -92,28 +92,27 @@ export class WriteLinkedCasesFieldComponent extends AbstractFieldWriteComponent
     }
   }
 
-  public getLinkedCaseReasons(serviceId: number): void {
+  public getLinkedCaseReasons(serviceId: number | string): void {
```

Comment:
Does this function actually return a value? I can't see where it does. It should either have a default return, or  should be void.

Reply:
The function doesn't return a value so its a void.
I've removed the additional type of 'number' from serviceID as the function is only called in this file and should always be passed as a string